config/db/easyfood_database.sql

```diff
@@ -123,3 +123,18 @@ CREATE TABLE IF NOT EXISTS `reviews` (
 INSERT INTO `reviews` (`id_usuario`, `id_restaurante`, `nota`, `comentario`) VALUES
 	(1, 1, 5, "Melhor restaurante da região."),
 	(3, 2, 3, "Bons pratos, péssimo atendimento.");
+									    
+CREATE TABLE IF NOT EXISTS `dim_cidade` (
+  `id` INT(11) unsigned NOT NULL AUTO_INCREMENT,
+  `codigo` INT(7) unsigned NOT NULL DEFAULT 0,
+  `nome` CHAR(255) NOT NULL DEFAULT '0',
+  `uf` CHAR(2) NOT NULL DEFAULT '0',
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=UTF8;
+									    
+LOAD DATA INFILE 'cidades.csv'
```

Comment:
Altera o path do arquivo para esse : 
```suggestion
LOAD DATA INFILE 'easyfood/cidades.csv'
```